### DISCUSSION TITLE
Printing with nylon - flsun-qq pro

### DISCUSSION THREAD

**Discussion by tubaian** [blackfyre/flsun.community#27] [Q&A]
Hi, I have been asked to make a replacement for a nylon gear wheel and was wondering if anyone on here has any experience of printing with nylon filament. Thanks, Ian

**Comment by blackfyre** [comment]
All metal hotend & enclosure... then you've got a chance at printing with nylon :D 
You might also need to tweak the firmware to go over 250 °C
Also, you might need to replace the spacers under the bed... mine were made of plastic and I'm not sure that it could withstand 100 °C for a long time...

**Comment by tubaian** [comment]
Thanks, I’m going to see if you will work before I alter my machine drastically.

Ian

Sent from my iPad

On 14 Jan 2022, at 08:17, Meki ***@***.***> wrote:

﻿

All metal hotend & enclosure... then you've got a chance at printing with nylon :D
You might also need to tweak the firmware to go over 250 °C
Also, you might need to replace the spacers under the bed... mine were made of plastic and I'm not sure that it could withstand 100 °C for a long time...

—
Reply to this email directly, view it on GitHub<https://github.com/blackfyre/flsun.community/discussions/27#discussioncomment-1967897>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AJEUTNC5K6IACAETF7Q3OGLUV7LZVANCNFSM5L2EQ2CA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
